operator/storage.go

```diff
@@ -57,7 +57,7 @@ func (s *storage) ListOperators(from int64, to int64) ([]registrystorage.Operato
 }
 
 func (s *storage) CleanRegistryData() error {
-	return s.cleanSyncOffset()
+	return s.db.RemoveAllByCollection(prefix)
```

Comment:
The operator private key is also under this prefix, you need to clear the sync offset prefix and operators prefix